01/style.css

```diff
@@ -19,5 +19,7 @@
     height: 0;
     border: 150px solid #dcdcdc;
     border-right-color: transparent;
-    border-bottom-color: transparent;    
+    border-bottom-color: transparent;
+    shape-outside: polygon(0 0, 0% 100%, 100% 0);
```

Comment:
wystarczy sam `shape-outside`.
Tutaj znajdziesz info o różnicy: 
https://stackoverflow.com/questions/55090634/difference-between-the-clip-path-and-shape-outside-properties